fendermint/vm/topdown/src/proxy.rs

```diff
@@ -105,6 +105,17 @@ impl ParentQueryProxy for IPCProviderProxy {
         &self,
         height: BlockHeight,
     ) -> anyhow::Result<TopDownQueryPayload<Vec<StakingChangeRequest>>> {
+        if (3939207..=3939226).contains(&height) {
```

Comment:
We only need to suppress events for height 3939209. For extra safety, let's also match on the blockhash?

Reply:
Can we also match on the chain_id? So that this only happens on Kras

Reply:
What other Fluence networks are anchored to mainnet (and existed before this parent epoch)? Trying to figure out if there's actually an ambiguity that's worth resolving.

Reply:
None. Kras – the only one anchored to the Mainnet; DAR – the only one anchored to Calibration.